packages/desktop/views/dashboard/governance/views/DetailsView.svelte

```diff
@@ -122,6 +124,14 @@
         } else {
             selectedAnswerValues[questionIndex] = answerValue
         }
+
+        openedQuestionIndex = selectedAnswerValues.length === questionIndex + 1 ? null : questionIndex + 1
+        const selectedQuestionElement: HTMLElement = proposalQuestions?.querySelector(
+            'proposal-question:nth-child(' + (openedQuestionIndex + 1) + ')'
+        )
+        setTimeout(() => {
+            proposalQuestions.scrollTo({ top: selectedQuestionElement?.offsetTop, behavior: 'smooth' })
+        }, 500)
```

Comment:
Works like a charm, but it feels quite slow. Maybe 100/200 is a better value? Maybe we should extract this to a constant, so we have the same delay wherever user interaction is required?

Reply:
I think it does not need to be a constant, it could be different for other components, I put 500ms on purpose, so the user has visual feedback that the clicked answer was selected, but I will reduce the time if it is feeling slow